pkg/controller/job/job_controller.go

```diff
@@ -448,6 +448,17 @@ func (jm *Controller) deletePod(logger klog.Logger, obj interface{}, final bool)
 	jm.enqueueSyncJobBatched(logger, job)
 }
 
+func (jm *Controller) addJob(logger klog.Logger, obj interface{}) {
+	jm.enqueueSyncJobImmediately(logger, obj)
+	jobObj, ok := obj.(*batch.Job)
+	if !ok {
+		return
+	}
+	if controllerName := managedByExternalController(jobObj); controllerName != nil {
+		metrics.JobByExternalControllerTotal.WithLabelValues(*controllerName).Inc()
```

Comment:
With beta -> alpha, do we still want metrics?

Reply:
Metrics are graduation criteria for beta, but they are ok to be included in alpha. 

The new metric was reviewed in the KEP, and didn't seem controversial, so I think it makes sense to keep it. 

It also helps a little to optimize the integration test, so that we can wait shorter amount of time before validating the update status didn't happen by the built-in controller.